subcommands/keys/ca_create.go

```diff
@@ -97,43 +95,35 @@ func doCreateCA(cmd *cobra.Command, args []string) {
 	resp, err := api.FactoryCreateCA(factory)
 	subcommands.DieNotNil(err)
 
-	writeFile("ca-csr", resp.CaCsr, 0400)
-	writeFile("tls-csr", resp.TlsCsr, 0400)
+	writeFile(x509.OnlineCaCsrFile, resp.CaCsr, 0400)
+	writeFile(x509.TlsCsrFile, resp.TlsCsr, 0400)
 	writeFile("create_ca", *resp.CreateCaScript, 0700)
 	writeFile("create_device_ca", *resp.CreateDeviceCaScript, 0700)
 	writeFile("sign_ca_csr", *resp.SignCaScript, 0700)
 	writeFile("sign_tls_csr", *resp.SignTlsScript, 0700)
 
 	fmt.Println("Creating offline root CA for Factory")
-	run("./create_ca")
+	resp.RootCrt = x509.CreateFactoryCa(factory)
```

Comment:
This sets the same bits as the previous implementation at lines 133-135, right?
Just asking to double check.

Reply:
Correct. In the case of the bash implementation, there is a `return string(readFile(FactoryCaCertFile))` inside CreateFactoryCa.